src/commands/platforms/create.ts

```diff
@@ -143,15 +171,24 @@ export default class PlatformCreate extends BaseCommand {
         message: 'What type of platform would you like to register?',
         choices: [
           'kubernetes',
+          agent_display_name,
+          //...(this.app.config.environment !== ENVIRONMENT.PRODUCTION ? [agent_display_name] : []),
```

Comment:
remove?

Reply:
This is temporary since the idea is to hide it in prod for now. The only reason I did this was for when people pulled the branch tomorrow for testing.